pl_bolts/models/detection/__init__.py

```diff
@@ -2,6 +2,9 @@
 
 try:
     from pl_bolts.models.detection.faster_rcnn import FasterRCNN
+    from pl_bolts.models.detection.detr import detr_model
+    from pl_bolts.models.detection.detr import detr_loss
+    from pl_bolts.models.detection.detr import detr_utils
```

Comment:
what is the reason for this wrapping?

Reply:
I didn't get you @Borda. I just tried to Match FRCNN and pass CI tests.
May I know what is the right way of doing this?
Since Detr needed 3 files I moved them to a folder, Let me know the changes.

Reply:
yes, but you wrapped the three imports in try/catch even only one is using `torchvision` so it would be better to isolate this import as much as possible, I 'll check it directly what we can do here...

Reply:
K. I will isolate it.

Reply:
@oke-aditya the try catch is to keep models free of deps :)

Reply:
Yes, the only 2 dependencies for `detr` are 
1. `scipy` in `detr_loss` it has no workaround.
2. `torchvision`, for its useful operations, used internally.

Reply:
I guess your commit solved this. Really thanks (I will make sure I don't mess it next time)

Reply:
Scipy is fine, it is everywhere, the same as NumPy... :]

Reply:
Marking as resolved 👍  Thank you for awesome help